students/minwookjang/users/views.py

```diff
@@ -50,9 +53,12 @@ def post(self, request):
         data = json.loads(request.body)
 
         try:
-            if not User.objects.filter(email = data['email'], password = data['password']):
+            if not User.objects.filter(email = data['email']):
                 raise ValidationError('INVALID_USER')
             
+            if not bcrypt.checkpw(data['password'].encode('utf-8'), User.objects.filter(email = data['email']).get().password.encode('utf-8')):
+                raise ValidationError('INVALID_USER')
```

Comment:
로직은 맞으나 가독성을 조금 더 좋게 하기 위해 다음과 같이 수정하면 좋을 것 같습니다.
```suggestion
            user = User.objects.get(email=data['email'])
            
            if not bcrypt.checkpw(data['password'].encode('utf-8'), user.password.encode('utf-8')):
                raise ValidationError('INVALID_USER')
```